code/modules/telesci/telesci_computer.dm

```diff
@@ -49,10 +49,10 @@
 /obj/machinery/computer/telescience/Initialize()
 	..()
 	for(var/i = 1; i <= starting_crystals; i++)
-		crystals += new /obj/item/ore/bluespace_crystal/artificial(null) // starting crystals
+		crystals += new /obj/item/stack/ore/bluespace_crystal/artificial(null) // starting crystals
```

Comment:
This entire thing is going to misbehave because the code expects each crystal to be one crystal, when it could actually be up to `max_amount` of bluespace_crystal.